packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/api/foo/index.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { createFoo, Client, FooClientOptions } from "./FooContext.js";
+export { Resource } from "./models.js";
```

Comment:
need to figure out why custom page is missing.

Reply:
temporarily disable the removeUnusedInterfaces as in this case, CustomPage is not being used in bar sub client, however it has been used in foo sub client. and removeUnusedInterfaces would just remove all of them, and I am not very sure how to fix that.